src/crimson/os/seastore/root_block.h

```diff
@@ -38,6 +38,8 @@ struct RootBlock : CachedExtent {
 
   root_t root;
 
+  std::map<std::string, std::string> meta;
```

Comment:
might want to use `std::map<string_view, string_view> meta`? as the content of `meta` *can* be backed by `root_t::meta[]`.

Reply:
No, keep it simple.

Reply:
Actually, we don't want to be decoding/encoding this every time we write out the root node.  Can we just keep this in memory as a char[] as with root_t?